### DISCUSSION TITLE
EdgeTX vs OpenTX

### DISCUSSION THREAD

**Discussion by meerfrau** [EdgeTX/edgetx#4236] [General]
Is EdgeTX a fork of OpenTX or the other way round? Who has forked and for which reason?

**Comment by pfeerick** [comment]
EdgeTX is a fork of OpenTX. It forked as development on OpenTX has essentially halted, and features that were promised to manufacturers (such as touch screen support) were non-existent over two years after commitments had been made to implement it. So it was forked and some old and some new developers have been working on it since, with support from various manufacturers and community. It doesn't appear as a fork in GitHub due to a limitation of  whereby you cannot fork a forked project to make PRs if you've forked the parent repo, so had to request for Github detach the fork so people could actually contribute to EdgeTX if they had also contributed to OpenTX (or wished to do so in the future if the situation was to change there).

**Reply by meerfrau** [reply]
Thanks for the info. When did the fork happen?

**Reply by gagarinlg** [reply]
Summer 2021

**Comment by meerfrau** [comment]
So this has been an Anti-FrSky fork?

**Reply by gagarinlg** [reply]
No, it was a
"We finally want touch screen support" 
fork.

And with OpenTX only one person decided the fate and had the right to merge PRs and commit to the main branch. Some people were fed up with this and the slow progress. After that one person went to FrSky OpenTX was pretty much dead, because there was no time left for OpenTX work.

**Reply by pfeerick** [reply]
I don't understand the question. How would it be an "Anti-FrSky fork" if we are continuing to support FrSky hardware, making it easier to support mods that let people use FrSky ISRM, and addding support for some for FrSkys latest receivers?

**Reply by raphaelcoeffic** [reply]
> So this has been an Anti-FrSky fork?

The classical answer to this type of question would be: "asked and answered".
